api/tokens.go

```diff
@@ -0,0 +1,179 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ONSdigital/dp-identity-api/emailvalidation"
+
+	"github.com/ONSdigital/log.go/log"
+)
+
+type ErrorStructure struct {
+	Errors []IndividualError `json:"errors"`
+}
+
+type IndividualError struct {
+	SpecificError string `json:"error"`
+	Message       string `json:"message"`
+	Source        Source `json:"source"`
+}
+
+type Source struct {
+	Field string `json:"field"`
+	Param string `json:"param"`
+}
```

Comment:
Thank you for pointing this out. 

Yes I think that it will live in a library but for now it was just decided to align the responses with the expected behaviour of the library.